src/app/components/dashboard/dashboard.html

```diff
@@ -1,120 +1,123 @@
+					<option value="1">1 sec</option>
+					<option value="2">2 sec</option>
+					<option value="3">3 sec</option>
+					<option value="5">5 sec</option>
+				</select>
+			</div>
+		</form>
+	</div>
 </div>
 <div class="row" ng-show="flags.isHostnameExpanded" ng-if="!vm.embed">
-    <div class="col-md-6">
-        <form role="form" name="form">
-            <div class="input-group input-group-lg target" ng-show="!flags.disableHostspecInput" ng-class="{'has-error': form.hostspec.$invalid && form.hostspec.$dirty}">
-                <span class="input-group-addon">Hostspec</span>
-                <input type="text" class="form-control" id="hostspecInput" name="host" data-content="Please enter the instance hostspec." rel="popover" data-placement="bottom" data-trigger="hover" data-container="body" ng-model="properties.hostspec" ng-change="vm.updateHost()" ng-model-options="{ updateOn: 'default', debounce: 1000 }" delay="1000" ng-disabled="flags.contextUpdating == true || ( flags.disableContainerSelectNone && flags.disableHostnameInputContainerSelect )" required placeholder="Instance Hostspec">
-            </div>
-            <div class="input-group input-group-lg target" ng-show="flags.enableContainerWidgets && !flags.disableContainerFilter">
-                <span class="input-group-addon">Container Filter</span>
-                <input type="text" class="form-control" id="globalFilterInput" name="globalFilter" data-content="Global filter for container widgets" rel="popover" data-placement="bottom" data-trigger="hover" data-container="body" ng-model="properties.containerFilter" ng-change="vm.updateContainerFilter()" ng-model-options="{ updateOn: 'default', debounce: 1000 }" delay="1000" placeholder="Container Filter">
-            </div>
-            <div class="input-group input-group-lg target" ng-show="flags.enableContainerWidgets && !flags.disableContainerSelect">
-                <span class="input-group-addon" >Container Id</span>
-                <select id="selectedContainer" ng-model="properties.selectedContainer" ng-change="vm.updateContainer()" class="form-control">
-                    <option ng-disabled="flags.disableContainerSelectNone && !flags.containerSelectOverride" value="">All</option>
-                    <option ng-repeat="container in properties.containerList">{{container}}</option>
-                </select>
-            </div>
-        </form>
-    </div>
-    <div class="col-md-6">
-        <p class="lead hostname-label" id="hostnameLabel" data-content="PMCD hostname. The hostname from the actual instance you're monitoring." rel="popover" data-placement="bottom" data-trigger="hover" data-container="body">{{flags.contextAvailable && properties.hostname ? properties.hostname : 'Disconnected'}}</p>
-    </div>
+	<div class="col-md-6">
```

Comment:
editorconfig states a indent of 2 spaces:
indent_style = space
indent_size = 2